about.html

```diff
@@ -0,0 +1,141 @@
+              <a class="nav-link px-3" href="#">News</a>
+              <a class="nav-link btn text-custom2 border-custom2 border-3 mx-3 d-none d-md-block" href="#">Jazzaholic Campaign</a>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section id="about-heading" class="bg-image bg-heading">
+        <div class="mask py-5 bg-custom7">
+          <div class="container-md">
+            <div class="row justify-content-center py-5">
+              <div class="col-11 pt-5 col-md-8">
+                <div class="row text-center justify-content-center">
+                  <h1 class="text-coco text-custom2 fw-bold display-1 display-3 mb-5">JAZZAHOLIC FESTIVAL 2021</h1>
+                  <h2 class="order-first fs-1 text-custom2 mb-3">"Hello! Jazz world"</h2>
+                  <p class="p-3 border-light border-2 bg-white mb-4">Jazzaholic Festival is a Colombian jazz event. The three day event sees the creatively unique line-up with artists from all over the world.</p>
+                  <p class="small mt-3 mb-0">
+                    Please contact us via email for any further questions about Jazzaholic Festival 2021!<br />
+                    <span class="heading-about-email">jazzaholic2021@gmail.com</span>
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="logo-section" class="py-5 border-bottom border-custom3 border-2">
+        <div class="container-md">
+          <div class="row text-center justify-content-center">
+            <div class="col-11 col-md-8">
+              <h2 class="text-coco display-6 mb-2">Jazzaholic Festival 2021 Logo</h2>
+              <div class="orange-line mx-auto mb-2"></div>
+              <p class="">The logo of Jazzaholic Festival was decided through the logo competition from June 8th to July 7th.</p>
+              <div class="py-3 px-5 border border-custom3 border-4 logo-section-logo mx-auto">
+                <img class="w-100" src="img/logo_jazzaholic-transparent.png" alt="logo-jazzaholic" />
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="past-events" class="py-5">
+        <div class="container-md">
+          <div class="row text-center justify-content-center">
+            <div class="col-11 col-md-10">
+              <h2 class="text-coco display-6 mb-2">See the past Jazzaholic Festivals</h2>
+              <div class="orange-line mx-auto mb-2"></div>
+              <p class="">Take a look at the last two Jazzaholic Festivals that took place in Bogotá.</p>
+              <div class="row text-white align-items-center py-md-5">
+                <a class="p-0 position-relative bg-custom2 col-md mb-3 mb-md-0 mx-md-2 hover">
+                  <img class="w-100 opacity-25" src="img/jazzaholic-2020.jpg" alt="jazzaholic-2020" />
```

Comment:
- [ ] OPTIONAL
Please don't leave any empty spaces in your code which do not serve any purpose